6tisch-roll-enrollment-priority.mkd

```diff
@@ -79,30 +79,32 @@ friendly router as a Join Proxy.
+They include available battery power,  already committed network bandwidth, and total available memory available for Neighbor Cache Entry (NCE) slots.
+An NCE entry is needed in order to maintain communication with the pledge.
 
 There are other situations where the operator of the network would like to selective enable or disable the enrollment process in a particular DODAG.
 
 As the enrollment process involves permitting unencrypted traffic into the best effort part of a (TSCH) network,  it would be better to have the enrollment process off when no new nodes are expected.
 
-A network operator might also be able to recognize when certain parts of the network are overloaded and can not accomodate additional enrollment traffic, and it would like to adjust the enrollment priority (the proxy priority field of {{!RFC9032}}) among all nodes in the subtree of a congested link.
-
-This document describes a RPL DIO option that can be used to announce a minimum enrollment priority.
+This document describes a RPL DIO option that can be used to set a minimum enrollment priority.
 The minimum priority expresses the (lack of) willingness by the RPL DODAG globally to accept new joins.
+
 It may derive from multiple constaining factors, e.g., the size of the DODAG, the occupancy of the bandwidth at the Root, the memory capacity at the DODAG Root, or an administrative decision.
 
 Each potential *Join Proxy* would this value as a base on which to add values relating to local conditions such as its Rank and number of pending joins, which would degrade even further the willingness to take more joins.
 
-When a RPL domain is composed of multiple DODAGs, nodes at the edge of 2 DODAGs may not only join either DODAG but also move from one to the other in order to keep their relative sizes balanced.
-For this, the approximate knowledge of size of the DODAG is an essential metric.
+When a RPL domain is composed of multiple DODAGs, nodes at the edge of 2 DODAGs may not only join either DODAG but also move from one to the other in order to keep their relative sizes balanced. For this, the approximate knowledge of size of the DODAG is an essential metric.
 Depending on the network policy, the size of the DODAG may or may not affect the minimum enrollment priority.
 It would be limiting its value to enforce that one is proportional to the other.
-This is why the current size of the DODAG is advertised separately in the new option.
+The current size of the DODAG is therefore also advertised in the new option.
 
 As explained in {{!RFC9032}}, higher values decrease the likelyhood of an unenrolled node sending enrollment traffic via this path.
 
 A network operator can set this value to the maximum value allowed, effectively disable all new enrollment traffic.
 
+Updates to this option propogate slowly through the network according to the trickle algorithm that periodically sends updates.
```

Comment:
If we assume that we never reset the DIO trickle timer, the option propagation in worst case may take hours or even days to propagate depending on DIO trickle timer parameters.